### PR TITLE
adds your mother

### PR DESCRIPTION
**Pull request by mc-oofert**
## About The Pull Request

adds a hallucination of the same rarity as a xeno leaping at you

https://github.com/tgstation/tgstation/assets/70376633/358dbf59-5add-40e6-ac53-693a971b2d61

your mother will have the same species as you

## Why It's Good For The Game

https://cddawiki.chezzo.com/cdda_wiki/index.php/Your_mother

## Changelog
:cl:
add: added a new hallucination, your mother
/:cl:

### CODE REVIEW

**Comment by Striders13** [comment]
you should change her first name to be random and last name taken from player

**Comment by mc-oofert** [comment]
> you should change her first name to be random and last name taken from player

felinids and humans only follow this convention

File: code/modules/hallucination/mother.dm [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+		return FALSE
+	var/turf/spawn_loc = pick(spawn_locs)
+	mother = new(spawn_loc, hallucinator, src)
+	mother.AddComponent(/datum/component/leash, owner = hallucinator, distance = get_dist(hallucinator, mother)) //basically makes mother follow them
+	point_at(hallucinator)
+	talk("[hallucinator]!!!!")
+	var/list/scold_lines = list(
+		pick(list("CLEAN YOUR ROOM THIS INSTANT!", "IT'S TIME TO WAKE UP FOR SCHOOL!!")),
+		pick(list("YOU INSULT YOUR GRANDPARENTS!", "USELESS!")),
+		pick(list("I BROUGHT YOU INTO THIS WORLD, I CAN TAKE YOU OUT!!!", "YOURE GROUNDED!!")),
```
Comment (Iamgoofball) [inline code/modules/hallucination/mother.dm:23]: *you're

Comment (mc-oofert) [reply code/modules/hallucination/mother.dm:23]: to be fair i have no clue how i typed the apostrophe in the first line in the first place

**Comment by Namwty** [comment]
Missed opportunity to use one of them massive clown abominations mobs as sprite.

**Comment by moocowswag** [comment]
> Missed opportunity to use one of them massive clown abominations mobs as sprite.

make it do that for clowns and people who have the honk moodlet?

**Comment by Fikou** [comment]
why does she speak like a robot

**Comment by EricZilla** [comment]
> why does she speak like a robot

Generational trauma

**Comment by san7890** [comment]
@Kylerace

**Comment by carlarctg** [comment]
She is not nearly wide enough to be your mother

**Comment by zxaber** [comment]
It's pretty funny, I'll give it that.

I don't think it's an interesting hallucination though. By and large pretty ignorable by the target, and worse, it screams to the player that they're hallucinating. Similar to known dead bodies in a morgue tray talking (but a whole lot less creepy).

**Comment by mc-oofert** [comment]
> it screams to the player that they're hallucinating.

it takes 3 seconds to differentiate between a hallucination and what is actually happening

**Comment by the-orange-cow** [comment]
There should be a permanent one of these as a brain trauma or wizard trauma or something.

**Comment by san7890** [comment]
oh dammit i just thought it through after i hit merge, will plasmaman images be on fire?

wait do plasmamen even have mothers?

what?

**Comment by CRITAWAKETS** [comment]
we'll just call it emergent gameplay and move on

**Pull request merged**